### PR TITLE
chore: reduce page size

### PR DESCRIPTION
**Pull request by omBratteng**
## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

AS-742

### Preview domain
https://as-742-reduce-page-size.preview.app.daily.dev

### CODE REVIEW

**Review by rebelchris — CHANGES_REQUESTED** [review]
This should be an experiment :)

File: packages/shared/src/lib/featureManagement.ts [hunk 1]
```diff
@@ -42,7 +42,15 @@ const feature = {
   ),
   extensionOverlay: new Feature('onboarding_extension_overlay', false),
   seniorContentOnboarding: new Feature('senior_content_onboarding', false),
-  plusSubscription: new Feature('plus_subscription', false),
+  plusSubscription: new Feature('plus_subscription', true),
```
Comment (rebelchris) [inline packages/shared/src/lib/featureManagement.ts:45]: Default is wrong here :) 
Also bottom one not sure if they match current?

File: packages/shared/src/components/FeedLayout.tsx [hunk 2]
```diff
@@ -1,73 +1,80 @@
...
+  | 'tablet'
+  | 'laptop'
+  | 'laptopL'
+  | 'laptopXL'
+  | 'desktop';
 
-export const feedBreakpoints = [tablet, laptop, laptopL, laptopXL, desktop];
+export type FeedSettings = {
+  pageSize: number;
+  breakpoint?: string;
```
Comment (rebelchris) [inline packages/shared/src/components/FeedLayout.tsx:22]: Why do we need the breakpoint in the settings?

File: packages/shared/src/components/FeedLayout.tsx [hunk 3]
```diff
@@ -80,39 +87,62 @@ const replaceDigitsWithIncrement = (str: string, increment: number): string => {
...
-  }
-  return reversedBreakpoints;
-};
 
 export default function FeedLayout({
   children,
 }: FeedLayoutProps): ReactElement {
   const { sidebarExpanded } = useContext(SettingsContext);
   const { sidebarRendered } = useSidebarRendered();
+  const feedPageSizes = useFeature(feature.feedPageSizes);
```
Comment (rebelchris) [inline packages/shared/src/components/FeedLayout.tsx:97]: We'll need to make sure this load appropriate in time, or we need to delay return maybe?

**Review by rebelchris — APPROVED** [review]
Code wise looks ok, let's maybe specifically test on preview after deployment?

File: packages/shared/src/components/FeedLayout.tsx (hunk 2 above)
Comment (omBratteng) [reply packages/shared/src/components/FeedLayout.tsx:22]: We don't really, just though it's cleaner to have it defined together with the rest, then a separate array like before

File: packages/shared/src/lib/featureManagement.ts (hunk 1 above)
Comment (omBratteng) [reply packages/shared/src/lib/featureManagement.ts]: woops, just had that for testing.

**Review by rebelchris — APPROVED** [review]
Nice a lot neater like this :)

**Pull request merged**